WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -147,7 +152,22 @@ final class BlogDashboardViewController: UIViewController {
     }
 
     private func addQuickStartObserver() {
-        NotificationCenter.default.addObserver(self, selector: #selector(loadCardsFromCache), name: .QuickStartTourElementChangedNotification, object: nil)
+        NotificationCenter.default.addObserver(forName: .QuickStartTourElementChangedNotification, object: nil, queue: nil) { [weak self] notification in
+
+            if let info = notification.userInfo,
+               let element = info[QuickStartTourGuide.notificationElementKey] as? QuickStartTourElement {
+
+                switch element {
+                case .setupQuickStart, .removeQuickStart:
+                    self?.loadCardsFromCache()
+                case .stats:
+                    self?.mySiteScrollView?.scrollToTop(animated: true)
+                    self?.mySiteViewController?.additionalSafeAreaInsets = UIEdgeInsets(top: 0, left: 0, bottom: Constants.bottomPaddingForQuickStartNotices, right: 0)
```

Comment:
Adding the `additionalSafeAreaInsets` ensures that the FAB is displayed above the notice 😄 

No additional insets | With additional insets
-- | --
<img src="https://user-images.githubusercontent.com/6711616/165064585-e3d4820c-1bc3-4730-b74c-22f6720aea6e.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/165064603-fc4364d8-54a3-4ff0-b657-d7ce2b0d37a2.png" width=200>


However I noticed not all QS tasks do this. I think it would be good to have consistency across all the tasks. I can clean this up / tackle this in a separate PR.